nautobot/dcim/templates/dcim/location.html

```diff
@@ -128,6 +128,14 @@
                 <td>{{ object.contact_email|hyperlinked_email }}</td>
             </tr>
         </table>
+        {% with request.path|add:"?tab=contacts"|urlencode as return_url %}
+            <div class="panel-footer text-right noprint">
+                <a href="{% url 'dcim:migrate_data_to_contact' pk=object.pk %}?return_url={{return_url}}" class="btn btn-primary btn-xs">
+                    <span class="mdi mdi-account-edit" aria-hidden="true"></span>
+                    Convert to contact/team
+                </a>
+            </div>
+        {% endwith %}
```

Comment:
Possibly wrap this in a `{% if request.user|has_perms:["extras.add_contactassociation"] %}` so that the button doesn't appear if the user lacks basic permissions?